setup.py

```diff
@@ -56,8 +56,8 @@ def read_requirements(filename):
     "smdebug_rulesconfig==1.0.1",
     "importlib-metadata>=1.4.0,<7.0",
     "packaging>=20.0",
-    "pandas",
```

Comment:
if we are adding pandas dependency as an extra, we might need to add documentation around using the extras install for use cases which involve using `pandas`.

Reply:
This I think will be a breaking change, customers who auto consume newest pysdk release with this fix would now need to also install pandas (in case they use it). But I do still think this is a valuable change considering the use-cases of supporting lambda.